### PR TITLE
Add Chakra UI community page as a direct link

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

**Comment by unicodeveloper** [comment]
@novoselrok @limitedmage Please can you review? I think a test is failing due to a prettier check. I need help with that as well. Thanks

File: client/web/src/Layout.tsx [hunk 1]
```diff
@@ -209,7 +209,7 @@ export const Layout: React.FunctionComponent<LayoutProps> = props => {
     ])
 
     // Hack! Hardcode these routes into cmd/frontend/internal/app/ui/router.go
-    const repogroupPages = ['/kubernetes', '/stanford', '/stackstorm', '/temporal', '/o3de', '/cncf']
+    const repogroupPages = ['/kubernetes', '/stanford', '/stackstorm', '/temporal', '/o3de', 'chakraui', '/cncf']
```
Comment (limitedmage) [inline client/web/src/Layout.tsx:212]:
```suggestion
    const repogroupPages = ['/kubernetes', '/stanford', '/stackstorm', '/temporal', '/o3de', '/chakraui', '/cncf']
```

**Review by limitedmage — APPROVED** [review]
Looks good. You can run `yarn prettier` locally to fix the prettier errors. Let me know if you need any help.

**Comment by unicodeveloper** [comment]
> Looks good. You can run `yarn prettier` locally to fix the prettier errors. Let me know if you need any help.

@limitedmage Please I need help with this. I ran into issues cloning and setting this up. Can you help run prettier on it?

File: client/web/src/repogroups/chakraui.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { SearchPatternType } from '@sourcegraph/shared/src/graphql-operations'
+
+import { RepogroupPage, RepogroupPageProps } from './RepogroupPage'
+import { RepogroupMetadata } from './types'
+
+export const chakraui: RepogroupMetadata = {
+    title: 'CHAKRA UI',
```
Comment (limitedmage) [inline client/web/src/repogroups/chakraui.tsx:9]: Is this supposed to be all-caps? It's not in other repogroup pages.

Comment (unicodeveloper) [reply client/web/src/repogroups/chakraui.tsx:9]: It should be 'Chakra UI' @limitedmage

Comment (limitedmage) [reply client/web/src/repogroups/chakraui.tsx:9]: @unicodeveloper FYI Looks like the wasn't fixed before the merge.

**Pull request merged**